.github/workflows/CI.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         arch:
           - x64
     steps:
-      - uses: actions/checkout@v2
```

Comment:
I've seen somewhere that there is also a bot that automatically makes a PR to update these GitHub actions. Not sure if it can also update itself :-).